libs/images/node-test/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM amidostacks/node:0.0.3
+
+RUN apt-get update -y && apt-get install dumb-init -y
```

Comment:
oh yeah and we don't need the dumb-init in this one as we are not going to have the `PID 1` in build containers